src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -141,6 +141,28 @@
         }
       }
     },
+    "generatorRegexMatch": {
+      "properties": {
+        "generator": {
+          "enum": [ "regexMatch" ]
+        },
+        "datatype":{
+          "enum": [ "bool" ]
+        },
+        "parameters": {
+          "properties": {
+            "source": {
+              "description": "The name of a different parameter in the template configuration. Its value will be used by this generator's regex to evaluate if there is a match. The value of the source parameter is not modified",
```

Comment:
"different symbol". Parameter is only one type of the symbols, assume any type can be used. "in the template configuration" can be probably omitted. 
I would suggest to short it down to "the symbol to search for a match".